app/controllers/home_controller.rb

```diff
@@ -1,4 +1,10 @@
 class HomeController < ApplicationController
   def index
+    @card = Card.ready_to_review
+      .order('RANDOM()')
+      .first
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.